src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1166,6 +1166,58 @@ internal virtual void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleB
         /// </summary>
         public abstract bool AreLocalsZeroed { get; }
 
+        internal virtual bool IsNullableEnabled() => false;
+
+        /// <summary>
+        /// If this method is a constructor, includes the nullable context for any
+        /// fields from the containing type that are set in this constructor.
+        /// </summary>
+        internal bool IsNullableEnabledCore(bool? isEnabledInBody)
+        {
+            if (isEnabledInBody == true)
```

Comment:
Does this literally mean "in the body of the method" vs. "in the body or signature"? The case I'm interested in is:

```c#
#nullable enable
override void M(string s = null)
#nullable disable
{
  s.ToString()
}
```

Essentially is the intent for `isEnabledInBody` to be `false` in the case above? If so how are we handling the nullable checks in the parameter and OHI? Is that calculated separately from this?

Reply:
`isEnabledInBody` should include the signature as well as the method body. I've renamed the parameter and added a comment. 

> If so how are we handling the nullable checks in the parameter and OHI? Is that calculated separately from this?

This method is used to determine the nullable context for `NullableWalker` only. Override signatures are checked separately.

---
In reply to: [545946391](https://github.com/dotnet/roslyn/pull/49953#discussion_r545946391) [](ancestors = 545946391)